modules/upgrade/partials/console/v3-breaking-changes.adoc

```diff
@@ -0,0 +1,18 @@
+- Authentication changes:
+
+** No more support for the plain login provider. Use a Redpanda SASL/SCRAM user instead.
```

Comment:
Probably worthwhile to mention that people need to enable impersonateUser to use SASL/SCRAM accounts.